conda/meta.yaml

```diff
@@ -10,42 +10,38 @@ source:
 build:
   noarch: generic
   script: python -m pip install --no-deps --ignore-installed .
-  script_env:
-    - PATH
-    - PYTHONPATH
-    - LD_LIBRARY_PATH
-    - OSPL_HOME
-    - PYTHON_BUILD_VERSION
-    - PYTHON_BUILD_LOCATION
-    - TS_CONFIG_OCS_DIR
 
 test:
   requires:
-    - pytest
+    - ts-conda-build =0.3
+    - pyside2
+    - asyncqt
+    - ts-idl {{ idl_version }}
+    - ts-salobj {{ salobj_version }}
   source_files:
     - python
     - bin
     - tests
-    - schema
     - setup.cfg
-    - .git
   commands:
-    - pytest --ignore tests/test_salpy_to_either.py --ignore tests/test_salobj_to_either.py
+    - pytest -vsx
 
 requirements:
   host:
     - python
     - pip
     - setuptools_scm
     - setuptools
-    - pytest-runner
-    - pytest-asyncio
-    - pytest-flake8
   run:
     - python
     - numpy
-    - astropy
```

Comment:
Can you please keep astropy in the list? I know it is required by ts-salobj, but leaving it included in the list (as bumpy is) is hopefully better for future, if ts_salobj decides to drop astropy dependency.